sentry_sdk/types.py

```diff
@@ -12,5 +12,10 @@
 
 if TYPE_CHECKING:
     from sentry_sdk._types import Event, Hint  # noqa: F401
+else:
+    # The lines below allow the types to be imported from outside `if TYPE_CHECKING`
+    # guards. The types in this module are only intended to be used for type hints.
+    Event = None
+    Hint = None
 
-    __all__ = ["Event", "Hint"]
+__all__ = ["Event", "Hint"]
```

Comment:
```suggestion
__all__ = ("Event", "Hint")
```

it doesn't really make much of a perf difference here but it's a good habit anyway